app/assets/stylesheets/base/_variables.scss

```diff
@@ -59,6 +59,8 @@ $success-button-color: $green;
 $success-hover-button-color: darken($green, 15);
 $warning-button-color: $yellow;
 $warning-hover-button-color: darken($yellow, 15);
+$danger-button-color: #df3e3e;
+$danger-hover-button-color: darken(#df3e3e, 15);
```

Comment:
Color literals like `#df3e3e` should only be used in variable declarations; they should be referred to via variable everywhere else.